### PR TITLE
Remove bok-choy usage

### PR DESCRIPTION
**Pull request by salman2013**
#Description

As the bok-choy has been deprecated https://github.com/openedx/public-engineering/issues/13, we are removing its usage from the code in this PR.

Ticket: https://github.com/orgs/openedx/projects/55/views/1?pane=issue&itemId=49965514

### CODE REVIEW

File: ecommerce/extensions/dashboard/refunds/tests/browser.py
Comment (feanil) [inline ecommerce/extensions/dashboard/refunds/tests/browser.py:1]: Looks like you got it working by moving this whole file over, do we need the whole file or are you planning on removing everything that you don't need now that you've got the tests passing?

Comment (salman2013) [reply ecommerce/extensions/dashboard/refunds/tests/browser.py:1]: @feanil Yes i have looked into few methods which are not required so i am going to remove them in the coming commits, The purpose of tagging was a preliminary review if this strategy looks fine? I have tried so many things with the selenium browser but the test was not passing due to some webdriver conflicts.

Comment (feanil) [reply ecommerce/extensions/dashboard/refunds/tests/browser.py:1]: yea, I think you're going in the right direction, let me know when this is ready for another round of review.  Hopefully we can drop most of the copied file but I'm okay with leaving it in if necessary.

Comment (salman2013) [reply ecommerce/extensions/dashboard/refunds/tests/browser.py:1]: @feanil It's ready for another round, i removed a few of the methods but some need to be there.

**Review by feanil — APPROVED** [review]

**Comment by feanil** [comment]
@salman2013 there are some conflicts on this branch, can you rebase and resolve them.  I think after that, this is good to merge.

**Comment by salman2013** [comment]
@feanil I have already resolved the conflicts and it seems its good to merge.

**Comment by feanil** [comment]
It still looks like it can't be merged to me: 
image

**Comment by salman2013** [comment]
@feanil could you please look at it again, thanks

**Pull request merged**